doc/release-notes/10800-add-dataverse-request-json-fix.md

```diff
@@ -0,0 +1 @@
+Fixed the "addDataverse" API endpoint (/dataverses/{id} POST) expected request JSON structure to parse facetIds as described in the docs.
```

Comment:
@GPortas heads up that I didn't add this to the 6.4 release notes because it pertains to a feature in #10644 that hasn't shipped yet (both will be in 6.4). From the perspective of the release notes, we don't need to mention bugs that never made it into a release.